Gopkg.toml

```diff
@@ -18,3 +18,8 @@
 [[constraint]]
   name = "github.com/pkg/errors"
   version = "0.8.0"
+
+[prune]
```

Comment:
let's not actually add anything to our `Gopkg.toml` until the implementation is there. lead by example, and all that 😄